README.md

```diff
@@ -167,6 +167,16 @@ if #available(iOS 13.0, *) {
 ```
 ***Note*** Flag is available on iOS 13 and above versions
 
+### Modify network requests
```

Comment:
Please add this to table of contents

Reply:
I would move this to the end of the readme, just before Known-Issues

Reply:
Done